### PR TITLE
Create fix_eth0_ip.sh

### PR DESCRIPTION
**Pull request by jimyang2008**
## Requirement
if there is ip address on eth0 interface, save it some where
if there is no ip address on eth0 for n seconds
  configure eth0 with last successful ip address manually
this is the dhcp over lay mode

### CODE REVIEW

**Comment by jerrchen** [comment]
possible to save also the default gateway and at least one dns?  if dhcp fails, we need to setup all of these.

File: scripts/fix_eth0_ip.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+ETH0_IP=$(ifconfig eth0 |grep 'inet addr'|awk '{print $2}' | awk -F: '{print $2}')
+
+SAVED_IP_FILE=/var/run/saved_ip
+
+if [[ -n "${ETH0_IP}" ]]
+then
+    /bin/rm -f ${SAVED_IP_FILE}
```
Comment (MelvinTo) [inline scripts/fix_eth0_ip.sh]: Need logic to wait for N seconds before using the saved ip.

Comment (jimyang2008) [inline scripts/fix_eth0_ip.sh]: wait for N seconds between *test* and *set* ? or just before *test* ?

Comment (MelvinTo) [inline scripts/fix_eth0_ip.sh]: both, test before and after.

Comment (jerrchen) [inline scripts/fix_eth0_ip.sh]: it is to ensure that 'dhcp' may be slow in some cases ...

**Pull request merged**